crates/ruff_python_parser/src/snapshots/ruff_python_parser__string__tests__backspace_alias.snap

```diff
@@ -1,19 +1,27 @@
 ---
 source: crates/ruff_python_parser/src/string.rs
 expression: suite
-snapshot_kind: text
```

Comment:
I always forget, is this because the snapshots haven't been touched in a while and so `cargo-insta` has removed this field on a recent version or is it that you've an older version of `cargo-insta`?